docs/site/Debugging-with-vscode.md

```diff
@@ -0,0 +1,18 @@
+---
+lang: en
+title: 'Debugging with VS Code'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, Debug
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Debugging-with-vscode.html
+---
+
+If you are using Visual Studio Code, below are some tips to help you debug your
+LoopBack applications.
+
```

Comment:
I think the main purpose of this page is to show how to debug a LoopBack application using VSCode. Could you please include this info?

Reply:
@dhmlau I am not sure other than the two tasks in `launch.json` file what else should be mentioned here, cuz debugging with vscode is not part of the LB knowledge. Any suggestions?

Reply:
What I meant is the instruction should be tailored to debugging a LoopBack application instead of LB4 framework. For example, if the scaffolded apps currently do not have `launch.json`, the instruction can tell the user to create such file with a provided content, etc.

Reply:
@dhmlau see https://github.com/strongloop/loopback-next/pull/6371#issuecomment-696811055, I can submit a separate PR for the vscode. I didn't realize the troubleshooting page is only for app developers. Thought it will cover debugging all the lb-next packages.